R/methods_gam.R

```diff
@@ -67,6 +67,11 @@ model_parameters.gam <- function(model,
   attr(parameters, "object_name") <- deparse(substitute(model), width.cutoff = 500)
   class(parameters) <- c("parameters_model", "see_parameters_model", class(parameters))
 
+  # converting CI to fraction
+  if ("CI" %in% names(parameters)) {
+    parameters$CI <- parameters$CI / 100
+  }
+
```

Comment:
This should actually be done in the related `ci()` method, which - if you caught all instances - should already return the fraction. To make maintenance easier and consistence, I would not add this `/ 100` code, but rather check that the `ci`-argument is properly set in the returned data frame.